rmw_fastrtps_shared_cpp/src/participant.cpp

```diff
@@ -158,7 +158,27 @@ rmw_fastrtps_shared_cpp::create_participant(
   }
 
   // Load default XML profile.
+  // Note: FASTRTPS_DEFAULT_PROFILES_FILE value is processed but it is a deprecated variable. Use FASTDDS_DEFAULT_PROFILES_FILE instead.
+  const char * env_value;
+  const char * error_str;
   auto factory = eprosima::fastdds::dds::DomainParticipantFactory::get_shared_instance();
+
+  error_str = rcutils_get_env("FASTRTPS_DEFAULT_PROFILES_FILE", &env_value);
+  if (error_str != NULL) {
+    RCUTILS_LOG_DEBUG_NAMED("rmw_fastrtps_shared_cpp", "Error getting env var: %s\n", error_str);
+    return nullptr;
+  }
+  if(env_value != nullptr) {
+    if (strcmp(env_value, "") != 0) {
+      RCUTILS_LOG_WARN_NAMED(
+        "rmw_fastrtps_shared_cpp",
+        "FASTRTPS_DEFAULT_PROFILES_FILE value is used for participant configuration, but it is deprecated and will no longer be supported. Use FASTDDS_DEFAULT_PROFILES_FILE instead.");
```

Comment:
```suggestion
        "FASTRTPS_DEFAULT_PROFILES_FILE value is used for participant configuration, "
        "but it is deprecated and will no longer be supported. "
        "Use FASTDDS_DEFAULT_PROFILES_FILE instead.");
```

linter does not warn this line? more than 100 cols in the single line?

Reply:
Changes applied on 1848ef2